FreeAPSWatch WatchKit Extension/Info.plist

```diff
@@ -3,11 +3,11 @@
 <plist version="1.0">
```

Comment:
What if we delete this file? Will the app pick [these translations automatically?](https://github.com/Artificial-Pancreas/iAPS/blob/dev/FreeAPS/Resources/Info.plist#L77-L80)  🤔

Reply:
I'm not sure, but I remember when it was needed to add this file, hence I would suggest not to delete it. Speaking of translations, you didn't add the English source strings for the localization.